msteams-platform/m365-apps/cards-loop-component.md

```diff
@@ -51,6 +46,14 @@ The following is a JSON example of an Adaptive Card-based Loop component with th
   "metadata": {
     "webUrl": "https://contoso.com/tab"
   },
+  "refresh": {
+    "action": {
+      "type": "Action.Execute",
```

Comment:
Do we have sample which showcases Action.Execute?

Reply:
Yes, @Wajeed-msft. See [here](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/msgext-unfurling-ac-loop-components/nodejs).